internal/plugin/adapter/common.go

```diff
@@ -4,22 +4,41 @@ import (
 	"context"
 
 	avngen "github.com/aiven/go-client-codegen"
+	"github.com/hashicorp/terraform-plugin-framework/datasource"
 	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
 )
 
-type DataView[T any] interface {
+// DatView datasource view interface
+type DatView[T any] interface {
```

Comment:
nit: mb `DSView`

Reply:
Don't know )
I tried to make them of equal length. Somehow it easier to read that way to me. If you don't have a strong opinion on that, I would keep this naming.